src/Debug/debugger.cpp

```diff
@@ -844,10 +848,22 @@ void Debugger::inspect(Module *m, uint16_t sizeStateArray, uint8_t *state) {
                 break;
             }
         }
+        fflush(stdout);
```

Comment:
is this flush of stdout really necessary? it is right in the middle of a json dump.

Reply:
Great. I'll approve the PR once you've removed the flush.